.github/actions/start-opensearch-with-one-plugin/action.yml

```diff
@@ -101,13 +101,13 @@ runs:
     # Verify that the server is operational
     - name: Check OpenSearch Running on Linux
       if: ${{ runner.os != 'Windows'}}
-      run: curl https://localhost:9200/_cat/plugins -u 'admin:admin' -k -v
+      run: curl https://localhost:9200/_cat/plugins -u 'admin:installPassword' -k -v
```

Comment:
Lets make a fast follow PR where this is passed into the action so the workflows can declare its own password instead of having yet another public password.